### PR TITLE
RBAC: Remove the username and class from the url  

### PR DESCRIPTION
**Pull request by fkovinAtRocket**
- /user/class/profile/level -> /profile/level
- use the authenticated user name from the request
- use the hardcoded class ZOWE

I've not yet replaced the call to ZIS with a direct RACROUTE call as this is a lower priority. 

Also, this PR contains an unrelated change that fixes the nasty problem with config file permissions in a dev sandbox.

### CODE REVIEW

File: c/zss.c [hunk 1]
```diff
@@ -795,6 +795,17 @@ static int validateConfigPermissionsInner(const char *path) {
   return 0;
 }
 
+#ifdef ZSS_IGNORE_PERMISSION_PROBLEMS
+
+static int validateFilePermissions(const char *filePath) {
```
Comment (1000TurquoisePogs) [inline c/zss.c]: The rest of the changes are OK, but can you please explain this? When/why is this needed? You just need permissions max 750 on the config dir and max 640 on the files in the config dir when running for the original check to pass. That's ../deploy/instance/ZLUX/serverConfig for the standard setup.

Comment (fkovinAtRocket) [reply c/zss.c]: Yeah, exactly, that was very annoying during development that I have to do that sort of thing with my sandbox. I often have to test in nonstandard environments (e.g. upload a minimal set of file to a system like SS03) and that all adds a level of headache. I'm going to do this anyway even if this is never committed.
